### PR TITLE
Documentation Update

### PR DESCRIPTION
**Pull request by julyate**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -56,7 +107,25 @@ klepto steal \
 --read-max-conns=8
 ```
 
-## Relationships
+Behind the scenes Klepto will establishes the connection with the source and target databases with the given parameters passed, and will dump 4 the tables.
```
Comment (julyate) [inline README.md]: Is this description correct for the example file?

File: README.md [hunk 2]
```diff
@@ -56,7 +107,25 @@ klepto steal \
...
+- `concurrency` to alleviate the pressure over both the source and target databases.
+- `read-max-conns` to limit the number of open connections, so that the source database does not get overloaded.
+
+<a name="configuration-file-options"></a>
+## Configuration File Options
+
+You can set the following fields in the file:
+* Tables
+* Matchers
+* more?
```
Comment (julyate) [inline README.md]: What other fields can you set in the toml file, and what do they mean?

File: README.md [hunk 3]
```diff
@@ -56,7 +107,25 @@ klepto steal \
 --read-max-conns=8
 ```
 
-## Relationships
+Behind the scenes Klepto will establishes the connection with the source and target databases with the given parameters passed, and will dump 4 the tables.
+
+We recommend to always set the following parameters:
```
Comment (rafaeljesus) [inline README.md]: 👍

File: README.md (hunk 1 above)
Comment (rafaeljesus) [reply README.md]: Hm I just realised that for this particular example we don't need to set
```
--concurrency
--read-max-conns
--read-max-idle-conns
```

Given this config only dumps a single table (users). We could move them to the recommend section below I think

File: README.md (hunk 2 above)
Comment (rafaeljesus) [reply README.md]: - `Tables` represents table definition
- `Filters` represents the way you want to filter the results
- `Matchers` are variables to replace placeholders in Filter
- `Relationships` represents a relationship definition

**Comment by rafaeljesus** [comment]
@julyate thanks a lot

**Pull request merged**